components/molecules/table/formatter/link.js

```diff
@@ -15,14 +15,12 @@ import { StyledLink } from '../../../'
 // Style
 import styled from 'styled-components'
 
-export const TableLink = (path, key, value, account) => ({ row }) => {
-  const getPath = () => {
-    let url = path
-
-    if (account) {
```

Comment:
@Ali-sharafi-git Have you tested this exclusion with the pages on portal that swap the Customer / Supplier and Admin links, as this i sremoved is there not a lot of refactor on the portal required?

Reply:
account parameter is not passed to this component in most places in portal.so path will be used.
I tested accounts, users, customers, suppliers list pages.
Portal code to change account list and user list is in [this ](https://github.com/DryKISS/portal.cleverly.works/pull/268/files)PR